### PR TITLE
N52: egui_ratatui

### PR DESCRIPTION
**Pull request by gold-silver-copper**
_Part of #1523_

### CODE REVIEW

File: content/news/052/index.md [hunk 1]
```diff
@@ -78,6 +78,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+![egui_ratatui running in bevy](egui_ratatui.jpg)
+
+[egui_ratatui][egui_ratatui] by [gold-silver-copper][gold] 
+- is an [egui][egui] widget that is also a [ratatui][ratatui] backend.
+- It allows you to create Terminal User Interfaces inside Egui.
+- Can be used with bevy, macroquad, eframe, pixels, etc
+- It is also WASM compatible.
+- The current release is the product of months of iteration, and is now "stable".
+- It is currently being used for a game and at a startup, with no issues.
```
Comment (Vrixyz) [inline content/news/052/index.md:91]: `a game and at a startup` -> I'm curious about that, which game, which startup ?

Comment (gold-silver-copper) [reply content/news/052/index.md:91]: @Vrixyz  The game is a roguelike in the interslavic language. The startup is GraphicsGrid Interactive Education, of which I am a member, we create educational software in various disciplines including language learning, geography, and geology. Our CEO is currently a FAANG engineer with experience in multiple other startups.

Thank you for your curiosity.

**Comment by janhohenheim** [comment]
Made an editorial pass. I'll merge it after the CI passes, but feel free to comment if you don't like something and I'll fix it in a follow-up PR :)

Feel free to also link your game once it's public!

**Review by janhohenheim — APPROVED** [review]

**Comment by gold-silver-copper** [comment]
@janhohenheim Thank you for the editorial pass, I think you did a great job. My name is not a Gödel, Escher, Bach reference, although that is a fantastic book. My name is based on my fascination with these 3 metals, due to their conductivity, shininess, and many other interesting chemical properties. I think they are super cool.

Happy programming and publishing !

**Pull request merged**